include/electionguard/precompute_buffers.hpp

```diff
@@ -17,129 +17,183 @@
 namespace electionguard
 {
     /// <summary>
-    /// This object holds the Triple for the entries in the precomputed triple_queue
-    /// The three items contained in this object are a random exponent (exp),
-    /// g ^ exp mod p (g_to_exp) and K ^ exp mod p (pubkey_to_exp - where K is
-    /// the public key).
+    /// A PrecomputedEncryption is a triplet-set of precomputed values
+    /// that are used to speed up encryption.
+    ///
+    /// It is used when encrypting a selection as an intermediate value
+    /// and it is used when generating a proof for a selection.
+    ///
+    /// The three items contained in this object are
+    /// - a random secret exponent r (secret - where r is [0,Q) and Q is the small prime modulus),
+    /// - g^r mod p (pad - where g is the generator and p is the large prime modulus)
+    /// - K^r mod p (blindingFactor - where K is the public key).
     /// </summary>
-    class EG_API Triple
+    class EG_API PrecomputedEncryption
```

Comment:
i renamed and refactored this to make it easier for a non-crypto person to reason about what is happening since the names were no longer valid so they had to be changed to something anyway.